datafusion/sql/src/parser.rs

```diff
@@ -44,6 +44,35 @@ fn parse_file_type(s: &str) -> Result<String, ParserError> {
     Ok(s.to_uppercase())
 }
 
+/// DataFusion specific EXPLAIN (needed so we can EXPLAIN datafusion
```

Comment:
This is the key thing -- we need a datafusion specific `EXPLAIN` statement

Reply:
Is there a reason to keep these datafusion specific statements in general vs. moving this upstream to the sqlparser crate? This distinction was difficult for me to understand when first diving into how statements are parsed.

Reply:
I think the high level rationale is that sqlparser-rs intends to parse sql for one or more existing "standard" sql dialects (like MySQL or Postgres). However, these statements (`COPY` and `CREATE EXTERNAL TABLE` are DataFusion specific extensions). I will make a PR with some docs to to try and clarify this rationale